hubblestack/daemon.py

```diff
@@ -457,6 +461,18 @@ def load_config():
                                         __opts__['log_level'],
                                         max_bytes=__opts__.get('logfile_maxbytes', 100000000),
                                         backup_count=__opts__.get('logfile_backups', 1))
+    logging.SPLUNK = 25 # logs below warning, above info
+    logging.addLevelName(logging.SPLUNK, 'SPLUNK')
+    def splunk(self, message, *args, **kwargs):
+        if self.isEnabledFor(logging.SPLUNK):
+            self._log(logging.SPLUNK, message, args, **kwargs)
+    logging.Logger.splunk = splunk
+    if __salt__['config.get']('hubblestack:splunklogging', False):
+        root_logger = logging.getLogger()
+        handler = hubblestack.splunklogging.SplunkHandler()
+        handler.setLevel(logging.SPLUNK)
+        root_logger.addHandler(handler)
+
```

Comment:
Is it possible to only enable timing information in splunklogging and not enable errors or warnings? Also, can I configure splunklogging for errors only and not for warnings? I am asking because I am interested in timing data, but not sure If I need all the warnings in Splunk.

Reply:
I think you want the errors and warnings in splunk -- even if you ignore them most of the time I think they are going go be useful to have when debugging issues for your customers. Any that explicitly aren't useful, like the insecure warnings that we had before we enabled SSL verify should be cleaned up as we see them.

If you decide that you don't want that data, we would need to make the `logging.SPLUNK` configurable and then we could set it to a value higher than ERROR so that logging at that level doesn't include anything else. Actually, that's a useful feature anyway, so I think I'll add it today anyway.